### ISSUE TITLE
Freeflycam SSE may not be working as intended, or may not be loaded with this version of SKSE

### ISSUE THREAD

**Issue by dervishxgit** [ForgottenGlory/Masterstroke#4]
From what I can tell, Freeflycam SSE is not working properly. 

Issue: Many keys unresponsive. Q and E do not roll, mouse wheel does not adjust FOV. WASD does work, but at a rapidity that requires SUCSM adjustment. Mouse L and R do move camera up and down.

Forum link [http://enbseries.enbdev.com/forum/viewtopic.php?f=6&t=5386&sid=4bb0075b53d296a7a1d1c0b71596a1b0&start=220] in readme indicates we may not be alone in experiencing this problem. It may simply be a result of which version of skse/skyrim it is compiled for. 

I do not see the plugin being loaded, I think it should show up in the skse64 log if it was:
[skse64.log](https://github.com/ForgottenGlory/Masterstroke/files/7893006/skse64.log)

I will continue to test, and try remapping the keys, to see if any different result can be produced.

**Comment by dervishxgit** [comment]
Ok, I think I figured it out. By default, the FreeFlyCam included with the list is not in the right folder structure. I reinstalled with the SKSE parent level directory and it seems to work. :))

Old:
\modFolder\FreeFlyCam*

New (working):
\modFolder\SKSE\plugins\FreeFlyCam*
![image](https://user-images.githubusercontent.com/2028096/150690542-eb9e988f-9cc8-4ed0-a7d7-1fb58a06f66a.png)


Sorry I didn't catch this sooner.
